.github/workflows/build.yml

```diff
@@ -42,13 +42,11 @@ jobs:
       # (e.g. - apt-get: gettext, etc; pip: circuitpython-build-tools, requirements.txt; etc.)
       run: |
         source actions-ci/install.sh
-    - name: Pip install pylint, Sphinx, pre-commit
+    - name: Pip install Sphinx, pre-commit
       run: |
-        pip install --force-reinstall pylint Sphinx sphinx-rtd-theme pre-commit
+        pip install --force-reinstall Sphinx sphinx-rtd-theme pre-commit
     - name: Library version
       run: git describe --dirty --always --tags
-    - name: Setup problem matchers
-      uses: adafruit/circuitpython-action-library-ci-problem-matchers@v1
```

Comment:
what's the consequence of moving this step?

Reply:
I don't really know.  I think what I figured was that it was added to cookiecutter as well as patched, and it just resulted in it being in 2 different places, so I figured picking one as standard would be a good idea.  But to be honest I don't know the significance of this step (as in I don't know what it does haha).